src/Domain/CoordinateMaps/PolarAngle.cpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/PolarAngle.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <pup.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace domain::CoordinateMaps {
+
+template <typename T>
+std::array<tt::remove_cvref_wrap_t<T>, 1> PolarAngle::operator()(
+    const std::array<T, 1>& xi) const {
+  return {{acos(-xi[0])}};
+}
+
+std::optional<std::array<double, 1>> PolarAngle::inverse(
+    const std::array<double, 1>& theta) const {
+  return {{-cos(theta[0])}};
+}
+
+template <typename T>
+tnsr::Ij<tt::remove_cvref_wrap_t<T>, 1, Frame::NoFrame> PolarAngle::jacobian(
+    const std::array<T, 1>& xi) const {
+  using ReturnType = tt::remove_cvref_wrap_t<T>;
+  tnsr::Ij<ReturnType, 1, Frame::NoFrame> result{};
+  get<0, 0>(result) = 1. / sqrt(1. - square(xi[0]));
```

Comment:
Are we guaranteed never to call this at 1 or -1?

Reply:
I added a warning to the docs. In practice we never call this on the poles because we use Gauss-Legendre points in cos(theta).

Reply:
You should never divide by powers of `sin(theta)` .  Even though you don't actually have a point at the poles, you will get convergence problems as the points approach the poles as you increase resolution.  (This was a problem with the original version of SpEC's CCE for example.)

If you use Pfaffian derivatives you will find that you never need to divide by powers of `sin(theta)` because all the powers of `sin(theta)` will cancel out (although sometimes it might require a bit of algebra to eliminate them).